tests/measurements/test_measurements.py

```diff
@@ -430,3 +433,116 @@ def test_obs_diagonalizing_gates(self):
         expected_classes = [qml.PauliZ, qml.S, qml.Hadamard]
         for op, c in zip(res, expected_classes):
             assert isinstance(op, c)
+
+
+class TestSampleMeasurement:
+    """Tests for the SampleMeasurement class."""
```

Comment:
Can we test method overriding for sample measurements as well? Just for future-proofing.

Reply:
Done!